packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -151,20 +170,20 @@ describe('common/sqlite', () => {
     expect(ctx.mocked['process.stdout.write'].mock.calls.join('\n')).toMatchInlineSnapshot(`
 
 
-      - Introspecting based on datasource defined in prisma/reintrospection.prisma
+                  - Introspecting based on datasource defined in prisma/reintrospection.prisma
 
-      ✔ Introspected 3 models and wrote them into prisma/reintrospection.prisma in XXXms
-            
-      *** WARNING ***
+                  ✔ Introspected 3 models and wrote them into prisma/reintrospection.prisma in XXXms
+                        
+                  *** WARNING ***
 
-      These models were enriched with \`@@map\` information taken from the previous Prisma schema.
-      - Model "AwesomeUser"
-      - Model "AwesomeNewPost"
-      - Model "AwesomeProfile"
+                  These models were enriched with \`@@map\` information taken from the previous Prisma schema.
+                  - Model "AwesomeUser"
+                  - Model "AwesomeNewPost"
+                  - Model "AwesomeProfile"
 
-      Run prisma generate to generate Prisma Client.
+                  Run prisma generate to generate Prisma Client.
 
-    `)
+            `)
```

Comment:
What caused this diff? I see only new test code.

Reply:
Unfortunately, Jest snapshots do that, we do not know how to avoid this at the moment 😢